frontends/PyCDE/src/pycde/module.py

```diff
@@ -2,7 +2,7 @@
 #  See https://llvm.org/LICENSE.txt for license information.
 #  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-from __future__ import annotations
+from .hacks import attribute_to_var
```

Comment:
Do we really need to name the module "hacks"?

Reply:
IMO `attribute_to_var` should live in `support`, alongside `var_to_attribute`, even if the implementation is considered hacky.

Reply:
Yeah, I thought about both. For the `hacks` name: I wanted to make it very clear that I consider this implementation to be _very_ hacky and should be replaced ASAP once the "correct" solution appeared. I chose to put it in pycde just to keep hacky stuff out of the bindings which none of the bindings code needs. (I'm thinking now that some of the stuff in `support.py` are kind of opinionated and should be moved into pycde.)

I don't feel strongly about either decision so if you have any opinion whatsoever, I'll move it.